app/views/admin/api_docs/base/_form.html.slim

```diff
@@ -9,8 +9,7 @@
     = form.input :system_name, input_html: { :disabled => true }
   = form.input :published, label: 'Publish?'
   = form.input :description, input_html: {rows: 3}
-  - unless @api_docs_service.new_record? && @api_docs_service.service_id.present?
-    = form.input :service_id, as: :select, collection: current_user.accessible_services
+  = form.input :service_id, as: :select, collection: current_user.accessible_services
```

Comment:
I think the conditional here was to ensure that, when you click on "New ActiveDoc" from the index of ActiveDocs of a service (therefore within the service section), we don't give the user the option to select a different service. I think this makes sense and we need to keep hiding the select box in this case.